### PR TITLE
docs: Add scroll behaviour smooth

### PR DESCRIPTION
**Pull request by amareshsm**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:
Added scroll behaviour to smoothly animate the scroll position.

Before:

https://user-images.githubusercontent.com/30730124/176533611-f7cfe020-b109-442b-b050-88c1a1b500c8.mov

After:

https://user-images.githubusercontent.com/30730124/176533661-50aa7ebb-eeeb-43a5-a694-12a3ba622ff4.mov

### CODE REVIEW

**Review by snitin315 — COMMENTED** [review]
Why is it needed? Is there an issue with the current scroll behavior?

**Comment by amareshsm** [comment]
> Why is it needed? Is there an issue with the current scroll behavior?

No issue with the current scroll behavior. Current scroll behavior (`auto`) scrolls to the position instantly so just set the value to `smooth` to scroll in a smooth fashion.

**Comment by snitin315** [comment]
I see. can you share a before/after screenshot/video?

**Comment by amareshsm** [comment]
> I see. can you share a before/after screenshot/video?

Updated in the PR.

**Comment by snitin315** [comment]
I see other documentation websites are also using the same current behavior. Scrolling can be a problem if the page is too long. So, I'm more inclined towards the current behavior. 

<details>
<summary>   
<a href="https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-script/#strategies"> Gatsby </a>  </summary>


https://user-images.githubusercontent.com/46647141/176577677-453f1ee2-8266-4a33-8970-e456999a786f.mov


</details>

<details>
<summary>   
<a href="https://vuejs.org/guide/essentials/computed.html"> Vue </a>  
</summary>

https://user-images.githubusercontent.com/46647141/176577768-bc406202-10f1-4a3f-8c44-9c6cdcde55de.mov
</details>

<details>
<summary>   
<a href="https://webpack.js.org/concepts/module-federation/"> Webpack </a>  
</summary>

https://user-images.githubusercontent.com/46647141/176577961-fc99b510-c88a-47df-a51a-8526a479f44e.mov


</details>

**Comment by nzakas** [comment]
I don’t have a strong opinion either way. I wouldn’t mind merging this to just see what feedback we get. It’s easy enough to undo if we get negative feedback.

File: docs/src/assets/scss/docs.scss [hunk 1]
```diff
@@ -1,5 +1,9 @@
 /* docs layout styles */
 
+html, body {
```
Comment (nzakas) [inline docs/src/assets/scss/docs.scss:3]: I think we only need html here 

```suggestion
html {
```

Comment (amareshsm) [reply docs/src/assets/scss/docs.scss]: done.

**Review by snitin315 — APPROVED** [review]
Let's try this.

**Review by mdjermanovic — APPROVED** [review]

**Pull request merged**